apps/vr-tests-react-components/src/stories/Avatar/Avatar.stories.tsx

```diff
@@ -185,81 +186,101 @@ const AvatarColors: React.FC<Pick<AvatarProps, 'active' | 'activeAppearance'>> =
+export default {
+  title: 'Avatar Converged',
+  component: Avatar,
+  decorators: [
+    story => (
+      <div style={{ display: 'flex' }}>
+        <div className="testWrapper" style={{ maxWidth: '750px' }}>
+          {story()}
+        </div>
       </div>
     ),
-    { includeRtl: true, includeHighContrast: true, includeDarkMode: true },
-  )
-  .addStory('size+name', () => <AvatarList names={examples.name} />)
-  .addStory('size+icon+badge+square', () => <AvatarList badge={{ status: 'out-of-office' }} shape="square" />)
-  .addStory('size+image+badge', () => <AvatarList images={examples.image} badge={{ status: 'do-not-disturb' }} />)
-  .addStory('size+inactive+badge', () => (
-    <AvatarList images={examples.image} active="inactive" badge={{ status: 'offline' }} />
-  ))
-
-  /* Temporarily disable these stories as these cause noise with storywright
-  The issue is raised against playwright. Till it gets fixed we disabled these. https://github.com/microsoft/playwright/issues/18373
-  .addStory('size+active+badge', () => (
-    <AvatarList images={examples.image} active="active" badge={{ status: 'available' }} />
-  ))
-  .addStory('size+active+shadow', () => (
-    <AvatarList images={examples.image} active="active" activeAppearance="shadow" />
-  ))
-  .addStory('size+active+ring-shadow', () => (
-    <AvatarList images={examples.image} active="active" activeAppearance="ring-shadow" />
-  ))*/
-  .addStory(
-    'badgeMask',
-    () => (
-      <div
-        style={{
-          backgroundSize: '32px 32px',
-          backgroundImage:
-            `repeating-conic-gradient(` +
-            `${tokens.colorBrandBackground} 0% 25%, ` +
-            `${tokens.colorBrandBackgroundSelected} 0% 50%)`,
-        }}
-      >
-        <AvatarList images={examples.image} color="marigold" active="active" badge={{ status: 'available' }} />
-      </div>
+    story => (
+      <StoryWright steps={new Steps().snapshot('normal', { cropTo: '.testWrapper' }).end()}>{story()}</StoryWright>
     ),
-    { includeRtl: true },
-  )
-  .addStory('customSize+image', () => <AvatarCustomSizeList images={examples.image} />)
-  .addStory('customSize+name+badge', () => (
-    <AvatarCustomSizeList names={examples.name} badge={{ status: 'available' }} />
-  ))
-  .addStory('customSize+icon+active', () => <AvatarCustomSizeList active="active" badge={{ status: 'available' }} />)
-  .addStory('color', () => <AvatarColors />, {
-    includeHighContrast: true,
-    includeDarkMode: true,
-  })
-  .addStory('color+active', () => <AvatarColors active="active" />, {
-    includeHighContrast: true,
-    includeDarkMode: true,
-  })
-  .addStory('image-bad-url', () => <Avatar name="Broken Image" image={{ src: `${imageRoot}/bad_image_url.jpg` }} />)
-  .addStory('image-bad-url+icon', () => <Avatar image={{ src: `${imageRoot}/bad_image_url.jpg` }} />);
+  ],
+} as Meta<typeof Avatar>;
```

Comment:
non-blocking question, won't be better here to have `satisfies Meta<typeof MyComponent>, wdyt? :)